src/screens/NFTSingleOfferSheet/index.tsx

```diff
@@ -307,37 +312,63 @@ export function NFTSingleOfferSheet() {
           }
           step.items?.forEach(item => {
             if (item.txHashes?.[0] && !txsRef.current.includes(item.txHashes?.[0]) && item.status === 'incomplete') {
-              let tx;
+              let tx: NewTransaction | null = null;
+              const asset = {
+                ...nft,
+                address: nft.asset_contract.address || '',
+                symbol: 'NFT',
+                decimals: 18,
+              };
               if (step.id === 'sale') {
                 tx = {
+                  status: 'pending',
                   to: item.data?.to,
                   from: item.data?.from,
                   hash: item.txHashes[0],
-                  network: offer.network,
-                  amount: offer.netAmount.decimal,
+                  network: offer.network as Network,
+                  nonce: item?.txHashes?.length > 1 ? nonce + 1 : nonce,
```

Comment:
idk why ts isn't complaining but couldnt `item?.txHashes?.length` be undefined here causing `item?.txHashes?.length > 1` to blow up

Reply:
this should still resolve as false